app/scripts/controllers/statefulSet.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+// TODO: statefulSetController rename
+// Kube docs: http://kubernetes.io/docs/user-guide/statefulSet/
+angular
+  .module('openshiftConsole')
+  .controller('StatefulSetController', function(
+    $filter,
+    $scope,
+    $routeParams,
+    AlertMessageService,
+    BreadcrumbsService,
+    DataService,
+    ProjectsService) {
+
+    $scope.projectName = $routeParams.project;
+    $scope.statefulSetName = $routeParams.statefulset || $routeParams.petset; // TODO: eliminate petset fallback
+    $scope.forms = {};
+    $scope.alerts = {};
+    // TODO: update getBreadcrumbs, currently returns:
```

Comment:
Now renders `Stateful Sets >> foo`, but clicking Stateful Sets isn't right because we don't have a Stateful Sets page.